src/AlgebraicGeometry/ToricVarieties/CohomologyClasses/special_attributes.jl

```diff
@@ -105,3 +105,111 @@ function intersection_form(v::NormalToricVariety)
     end
     return intersection_dict_for_user
 end
+
+
+@doc raw"""
+    chern_class(v::NormalToricVariety, k::Int; check::Bool = true)
+
+Computes the `k`-th Chern class of the tangent bundle of a normal toric variety
+that is both smooth and complete. Since these checks can be computationally
+very demanding, we provide an optional argument `check`. Once set to `false`,
+this method skips those tests.
+
+The implemented algorithm uses proposition 13.1.2 in [CLS11](@cite).
+
+# Examples
+```jldoctest
+julia> F3 = hirzebruch_surface(NormalToricVariety, 3)
```

Comment:
As long as `hirzebruch_surface` can only return `NormalToricVariety`, this argument is redundant.

Just in case you were asking: no, it does not make things globally more consistent.

Reply:
I have just added simplified constructors in https://github.com/oscar-system/Oscar.jl/pull/3996/commits/3889afd6ded369496c00df59ca65c247cd8b256f. This should (hopefully) fix this issue.

Reply:
It is not redundant here, as adding a default will keep this default necessary in all of Oscar 1.x.